.gitignore

```diff
@@ -11,6 +11,12 @@ thirdparty/installed
 *.so.tmp
 .DS_Store
 *.iml
+extension/spark-doris-connector/.classpath
+extension/spark-doris-connector/target
+loadManagerTest
```

Comment:
When and where we will generate  `loadManagerTest `

Reply:
when you run unit test in idea